test/legacy_test/op_test.py

```diff
@@ -1290,6 +1290,17 @@ def get_ir_input_attr_dict_and_feed(self, stop_gradient):
                 input_dict.update({name: x})
         return static_inputs, attrs_outputs, input_dict, feed
 
+    def _need_fetch(self, sig_name):
+        if sig_name in self.outputs.keys():
```

Comment:
```suggestion
        if sig_name in self.outputs:
```
self.outputs.key() 返回的应该是一个迭代器，查询效率是O(N)的，这里直接可以用关键字 in，效率是O(1)

Reply:
下个pr修改下，感谢~